src/gbo/GameBoyCore.js

```diff
@@ -7895,12 +7895,20 @@ GameBoyCore.prototype.memoryReadJumpCompile = function () {
 						return 0x8F | parentObj.memory[0xFF75];
 					}
 					break;
-				case 0xFF76:
-				case 0xFF77:
-					this.memoryHighReader[index & 0xFF] = this.memoryReader[index] = function (parentObj, address) {
-						return 0;
-					}
-					break;
+                case 0xFF76:
+                    //Undocumented realtime PCM amplitude readback:
```

Comment:
I need to fix regs 0x76 and x077 and the HDMA loop reading out of bounds in a rare case. I need to reopen issue tracking. :I